website/settings/defaults.py

```diff
@@ -280,3 +281,5 @@ def parent_dir(path):
 
 ENABLE_ARCHIVER = True
 ###########################
+
+MAX_INDEXED_FILE_SIZE = 5 * (10 ** 6)  # Megabytes
```

Comment:
Any reason we're using the SI megabyte (10<sup>6</sup>) instead of a binary megabyte (2<sup>20</sup>)? The former is ~5% smaller than the latter.